src/Common.jl

```diff
@@ -115,34 +115,37 @@ ls, lsf
 """
 sortDFG(vars::Vector{Symbol})::Vector{Symbol} = sortVarNested(vars)
 
-"""
-    $SIGNATURES
-
-Return the factor type used in a `::DFGFactor`.
-
-Notes:
-- OBSOLETE, use newer getFactorType instead.
-
-Related
-
-getFactorType
-"""
-function getfnctype(data::GenericFunctionNodeData)
-  # TODO what is this?
-  if typeof(data).name.name == :VariableNodeData
-    return VariableNodeData
-  end
-
-  # this looks right
-  return data.fnc.usrfnc!
-end
-function getfnctype(fact::DFGFactor; solveKey::Symbol=:default)
-  data = solverData(fact) # TODO , solveKey=solveKey)
-  return getfnctype(data)
-end
-function getfnctype(dfg::T, lbl::Symbol; solveKey::Symbol=:default) where T <: AbstractDFG
-  getfnctype(getFactor(dfg, exvertid))
-end
+#TODO Is this deprecate in favor of getFactorType, or OBSOLETE?
+# """
+#     $SIGNATURES
+#
+# Return the factor type used in a `::DFGFactor`.
+#
+# Notes:
+# - OBSOLETE, use newer getFactorType instead.
+#
+# Related
+#
+# getFactorType
+# """
+# function getfnctype(data::GenericFunctionNodeData)
+#   # TODO what is this?
+#   if typeof(data).name.name == :VariableNodeData
+#     return VariableNodeData
+#   end
+#
+#   # this looks right
+#   return data.fnc.usrfnc!
+# end
+# function getfnctype(fact::DFGFactor, solveKey::Symbol=:default)
+#   data = getSolverData(fact) # TODO , solveKey)
+#   return getfnctype(data)
+# end
+# function getfnctype(dfg::T, lbl::Symbol, solveKey::Symbol=:default) where T <: AbstractDFG
+#   getfnctype(getFactor(dfg, exvertid))
+# end
+
+@deprecate getfnctype(args...) getFactorType(args...)
```

Comment:
I just left the comments here for tracking. Can be deleted if merged and @deprecated moved.
The TODO bothered me a bit.